README.md

```diff
@@ -104,6 +104,7 @@ Thanks to all [contributors](https://github.com/ciconia/awesome-music/graphs/con
 * [Tunefl](https://github.com/tiredpixel/tunefl) - LilyPond mini-score engraving and sharing service for musicians.
 * [Verovio](https://github.com/rism-ch/verovio) - a library and a toolkit for engraving MEI music notation into SVG.
 * [Vexflow](https://github.com/0xfe/vexflow) - a JavaScript library for rendering music notation and guitar tablature.
+* [Unison](https://unisonofficial.com/html/user-documentation.html?title=unison-editor) - Proprietary software for creating music score with words.
```

Comment:
Hi, please maintain alphabetical order so I can merge this. Thanks!

Reply:
@ciconia done